lib/license_finder/package_managers/nuget.rb

```diff
@@ -89,6 +89,23 @@ def package_management_command
       "mono #{nuget_binary}"
     end
 
+    def prepare
+      _, stderr, status = Dir.chdir(project_path) { Cmd.run(prepare_command) }
+      return if status.success?
+      log_errors stderr
+      logger.info prepare_command, 'trying fallback prepare command', color: :magenta
+
+      cmd = "#{prepare_command} -PackagesDirectory ."
+      stdout, stderr, status = Dir.chdir(project_path) { Cmd.run(cmd) }
+      return if status.success?
+      log_errors_with_cmd(cmd,stderr)
+
+      error_message = "Prepare command '#{cmd}' failed\n#{stderr}"
+      error_message = error_message.concat("\n#{stdout}\n") if !stdout.nil? && !stdout.empty?
```

Comment:
You can try using `+` if you want to retain frozen_string_literal and see if that works

```suggestion
      error_message = error_message + "\n#{stdout}\n" if !stdout.nil? && !stdout.empty?
```

Reply:
Yes, that sounds like a good idea. We can give it a shot on Monday.